src/Graphics/src/Graphics.Gtk/Graphics.Gtk.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <TargetFrameworks>netstandard2.1;netstandard2.0;$(_MauiDotNetTfm)-gtk</TargetFrameworks>
     <RootNamespace>Microsoft.Maui.Graphics.Gtk</RootNamespace>
+    <PackageId>Mali.Graphics.Gtk</PackageId>
```

Comment:
@webwarrior-ws indentation seems different than rest of the file

Reply:
No, I checked in notepad++ with invisible symbols on, it's the same.